src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  const dailyRate = 40;
+  const longTermDiscount = 50;
+  const shortTermDiscount = 20;
+  const longTermThreshold = 7;
+  const shortTermThreshold = 3;
```

Comment:
For such variables we should use UPPERCASE. Using UPPERCASE for constant variables is a common convention in many programming languages to indicate that these values should not change during the execution of the program.
```suggestion
  const DAILY_RATE = 40;
  const LONG_TERM_DISCOUNT = 50;
  const SHORT_TERM_DISCOUNT = 20;
  const LONG_TERM_THRESHOLD = 7;
  const SHORT_TERM_THRESHOLD = 3;
```